syntax/string.js

```diff
@@ -0,0 +1,20 @@
+const String = {
+  /**
+   * The ucwords() method capitalizes the first letter of every word in a string.
+   * @param {string} str
+   * @returns {string}
+   * @example
+   * range("hello world") // output:"Hello World"
```

Comment:
the name of the `example` function is not compatible with the actual `function` just change it and you're good to go 👌🏻

Reply:
in the example, you still have the `range` function but the actual name of the function is `ucwords`.

Reply:
It's a copying problem, actually. I am using the same Doxygen as your array module.

Reply:
yes it is better for making doc usage of the code/functions

Reply:
that was amaizing